frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.module.css

```diff
@@ -77,7 +71,10 @@ body {
     transition: color 1s linear;
   }
 
+  /* A temporary solution until we migrated all colors in visualization to use semantic colors
+   e.g. `text-primary`, `text-secondary`, `text-tertiary`. */
   .EmbedFrame svg text {
+    fill: var(--mb-color-text-primary);
```

Comment:
This selector is already inside the dark theme selector, so it basically works the same way as

https://github.com/metabase/metabase/blob/dd00df1a06ae729d9641e147a0fc91bf54349923/frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.module.css#L32-L35